src/js/dropdown.ts

```diff
@@ -5,9 +5,13 @@ import { CityId, ScoreCardDetails, dropdownChoice } from "./types";
 
 export const DROPDOWN = new Choices("#city-dropdown", {
   allowHTML: false,
-  itemSelectText: "Select",
+  itemSelectText: "",
   searchEnabled: true,
-  shouldSort: false, // since cities are already alphabetical order in scorecard, disabling this option allows us to show PRN maps at the top.
+  searchResultLimit: 6,
+  searchFields: ["label"],
```

Comment:
The default was `["label", "value"]`, meaning it would be searching for city IDs like `santa-ana-ca`. We don't expect users to be searching by city ID.